src/opm/input/eclipse/EclipseState/Aquifer/Aquancon.cpp

```diff
@@ -218,13 +233,12 @@ namespace Opm {
     }
 
 
-    const std::vector<Aquancon::AquancCell>& Aquancon::operator[](int aquiferID) const {
-        const auto search = this->cells.find(aquiferID);
-        if (search == this->cells.end()) {
-            auto msg = fmt::format("There is no connection associated with analytical aquifer {}\n", aquiferID);
-            throw std::runtime_error(msg);
-        }
-        return search->second;
+    const std::vector<Aquancon::AquancCell>& Aquancon::getConnections(int aquiferID) const {
+        return this->cells.at(aquiferID);
+    }
```

Comment:
This will throw without  a nice error message. Intended?

Reply:
The original intention is that we should not call this function if there is no connection associated with aquifer with id `aquiferID`, and in its current form it is guarded from the outside when calling the function.  I almost think we can use `assert` here since I am considering it is a programming mistake. 

Now I think it might just be easier that we move the warning messages inside the function `Aquancon::getConnections`. Any suggestions?